docker/Build.plan

```diff
@@ -0,0 +1,4 @@
+FROM fpco/stack-build:lts-8.0
```

Comment:
switch to latest lts? also hard to keep up ... would be cool if this was dynamic upon creation